packages/cli/package.json

```diff
@@ -24,7 +24,8 @@
     ".": {
       "import": "./dist/index.js",
       "types": "./dist/index.d.ts"
-    }
+    },
+    "./package.json": "./package.json"
```

Comment:
Adding this so that we can `require.resolve('@shopify/cli/package.json')` later.
If for some reason this is not desired, we can use [`findPathUp` like we do in Hydrogen](https://github.com/Shopify/hydrogen/blob/6bc6068fbffebbcd07c3d1db97956248bfa6a742/packages/cli/src/lib/build.ts#L37-L62) (we needed that in Hydrogen because the package can be run globally bundled, or locally, so require.resolve wouldn't be enough).